test/e2e/dns.go

```diff
@@ -51,12 +51,10 @@ func createDNSPod(namespace, wheezyProbeCmd, jessieProbeCmd string) *api.Pod {
 		ObjectMeta: api.ObjectMeta{
 			Name:      "dns-test-" + string(util.NewUUID()),
 			Namespace: namespace,
-			Annotations: map[string]string{
```

Comment:
we should keep both cases for now, with a TODO to remove the annotations test case after 1.3